Automattic/wp-calypso
# Stats: Swapping Search and Countries components

#### Proposed Changes

* using a feature flag and replacing `Search` and `Countries` StatsModules with the recently created component with horizontal bars
* extending horizontal bar and stats card components to accept a hero item placed before the component's header (e.g. world map), allowing for items that don't have any click interaction
* adding a component to render country flags

#### Testing Instructions



* Navigate to the main Stats page either on a local instance or the test URL from the comment below
* append `?flags=stats/new-stats-module-component` to the URL
* verify that `Search` component has horizontal bars
* verify that `Countries` component has horizontal bars

#### Pre-merge Checklist






Related to #69044
